src/org/opendatakit/briefcase/export/ExportEvent.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (C) 2018 Nafundi
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package org.opendatakit.briefcase.export;
+
+public final class ExportEvent {
+  private final String formId;
+  private final String statusLine;
+
+  private ExportEvent(String formId, String statusLine) {
+    this.formId = formId;
+    this.statusLine = statusLine;
+  }
+
+  public static ExportEvent progress(FormDefinition form, double percentage) {
```

Comment:
I recommend some changes here to improve the presentation of these messages.

![screen shot 2018-05-13 at 10 41 48 am](https://user-images.githubusercontent.com/55099/39970091-765c8e32-569a-11e8-8fcf-28a0bd578355.png)

Let’s pluralize correctly. 

“Export operation” is longer and no better than “Export”.  Same for “Successfully exported” and “exported”, and the latter allows the whole message to fit in the GUI on my computer, as you can see below.

![screen shot 2018-05-13 at 10 40 40 am](https://user-images.githubusercontent.com/55099/39970105-c97121dc-569a-11e8-90fd-4776d85549a8.png)

I’m confused by the “Not exported yet”.

Reply:
Both of these statements can’t be true:
![screen shot 2018-05-17 at 1 41 11 pm](https://user-images.githubusercontent.com/55099/40202833-1eba4872-59d8-11e8-8e6f-6d4e7a8dd199.png)